### PR TITLE
Camera documentation

### PR DESCRIPTION
**Pull request by jabarragann**
Documentation about camera conventions:
1) Diagrams showing the different coordinate frames in the surgical challenge.
2) Explained differences between AMBF and openCV camera conventions.
3) Example script where all the concepts are put together. The script projects salient points of the needle into the image plane.

### CODE REVIEW

File: docs/camera_conventions.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Camera coordinate frames
+
+
+
+## Different coordinate frames for the cameras
+
+The following simulation includes a pair of stereo cameras that are fixed with respect to a camera frame. When commanding the virtual camera arm, the relation between the cameras and the frame will remain fixed, i.e. , transformations `T_FL` and `T_FR` do not change when moving the virtual ECM.  Finding a transformation between the needle local coordinate frame and the left image plane can be accomplish via the following block of code:
```
Comment (adnanmunawar) [inline docs/camera_conventions.md:7]: The following scene includes ...

File: docs/camera_conventions.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    T_CN_CV2 = F.dot(T_CN_AMBF)
+````
+
+
+
+<img src="./figures/camera_convention.png" width="640"  />
+
+
+
+A full example of how to use this transformation matrices to project points from the needle into the image can be found in [project_needle_points.py](./../scripts/surgical_robotics_challenge/examples/project_needle_pts.py)
```
Comment (adnanmunawar) [inline docs/camera_conventions.md:47]: how to use "these" transformation matrices ...

File: docs/camera_conventions.md (hunk 1 above)
Comment (adnanmunawar) [inline docs/camera_conventions.md:7]: The word simulation here might be confusing with the general AMBF simulation, may be better to use a different word

**Comment by adnanmunawar** [comment]
Thanks for the PR Juan. I added some minor comments.

**Comment by adnanmunawar** [comment]
Hi Juan, can you also add a section for the camera intrinsic parameters. I.e. how to get the intrinsic matrix from the Field View Angle, width and height of the camera.

**Comment by jabarragann** [comment]
No problem Adnan. Thanks for the comments. I will probably take care of the modifications during the weekend.

**Comment by jabarragann** [comment]
Hi Adnan, sorry for the delay. I already got all your corrections and added the intrinsic information. If you have any additional corrections I can do them today.

**Pull request merged**